packages/synapse-react-client/test/utils/hooks/usePreFetchResource.integration.test.tsx

```diff
@@ -36,14 +36,13 @@ describe('usePreFetchResource tests', () => {
       ),
     )
 
-    const { result, waitForNextUpdate } = renderHook(() =>
```

Comment:
waitForNextUpdate is not in @testing-library/react, but that's ok, we can just use `waitFor`